### PR TITLE
Add quantized model export description

### PR DESCRIPTION
**Pull request by linbinskn**

### CODE REVIEW

File: docs/en_US/Compression/QuickStart.rst [hunk 1]
```diff
@@ -194,8 +194,8 @@ Some compression algorithms use epochs to control the progress of compression (e
 
 ``update_epoch`` should be invoked in every epoch, while ``step`` should be invoked after each minibatch. Note that most algorithms do not require calling the two APIs. Please refer to each algorithm's document for details. For the algorithms that do not need them, calling them is allowed but has no effect.
 
-Export Compressed Model
-^^^^^^^^^^^^^^^^^^^^^^^
+Export Pruning Model
```
Comment (QuanluZhang) [inline docs/en_US/Compression/QuickStart.rst:197]: Pruning -> Pruned

File: docs/en_US/Compression/QuickStart.rst [hunk 2]
```diff
@@ -209,4 +209,27 @@ You can easily export the compressed model using the following API if you are pr
 
    pruner.export_model(model_path='model.pth', mask_path='mask.pth', onnx_path='model.onnx', input_shape=[1, 1, 28, 28])
 
+Export Quantized Model
+^^^^^^^^^^^^^^^^^^^^^^
+You can export the quantized model directly by using ``torch.save`` api and the quantized model can be loaded by ``torch.load`` without any extra modification. The following example shows the normal proceduce of saving, loading quantized model and get related parameters in QAT.
```
Comment (QuanluZhang) [inline docs/en_US/Compression/QuickStart.rst:214]: proceduce -> procedure

Comment (linbinskn) [reply docs/en_US/Compression/QuickStart.rst]: OK, fixed.

**Review by QuanluZhang — APPROVED** [review]

File: docs/en_US/Compression/QuickStart.rst [hunk 3]
```diff
@@ -194,8 +194,8 @@ Some compression algorithms use epochs to control the progress of compression (e
 
 ``update_epoch`` should be invoked in every epoch, while ``step`` should be invoked after each minibatch. Note that most algorithms do not require calling the two APIs. Please refer to each algorithm's document for details. For the algorithms that do not need them, calling them is allowed but has no effect.
 
-Export Compressed Model
-^^^^^^^^^^^^^^^^^^^^^^^
+Export Pruned Model
+^^^^^^^^^^^^^^^^^^^^
 
 You can easily export the compressed model using the following API if you are pruning your model, ``state_dict`` of the sparse model weights will be stored in ``model.pth``\ , which can be loaded by ``torch.load('model.pth')``. In this exported ``model.pth``\ , the masked weights are zero.
```
Comment (J-shang) [inline docs/en_US/Compression/QuickStart.rst:200]: `You can easily export the compressed model` -> `You can easily export the pruned model` ?

Comment (linbinskn) [reply docs/en_US/Compression/QuickStart.rst]: OK.

**Review by J-shang — APPROVED** [review]

**Pull request merged**